UnitTests/BTDropInControllerTests.swift

```diff
@@ -1,55 +0,0 @@
-import XCTest
-
-class BTDropInControllerTests: XCTestCase {
-    
-    func testInitWithAuthorization_createsThreeDSecureRequest_whenAmountIsSet_andThreeDSecureIsRequested() {
-        let dropInRequest = BTDropInRequest()
-        dropInRequest.amount = "12.54"
-        dropInRequest.threeDSecureVerification = true
-        
-        let dropInController = BTDropInController(authorization: "development_tokenization_key",
-                                                  request: dropInRequest,
-                                                  handler: nil)
-        
-        XCTAssertEqual(dropInController?.dropInRequest.threeDSecureRequest?.amount, NSDecimalNumber(string: "12.54"))
-    }
-    
-    func testInitWithAuthorization_doesNotCreateThreeDSecureRequest_whenThreeDSecureIsNotRequested() {
```

Comment:
Why are we removing this test? Isn't this testing `dropInRequest.threeDSecureVerification = false` and not `dropInRequest.amount`.

Reply:
Nevermind. Makes sense to get rid of this file!